include/TrustWalletCore/TWBravoAddressType.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+#include "TWBase.h"
+
+TW_EXTERN_C_BEGIN
+
+TW_EXPORT_ENUM(uint32_t)
+enum TWBravoAddressType {
+    TWBravoAddressTypeMainNet = 0,
+    TWBravoAddressTypeTestNet = 1,
```

Comment:
maybe we can remove testnet support? less code and less tests, because the mainnet is launched. we just removed binance testnet support

Reply:
Maybe we can remove after the integration with Front-end?